pkg/i18n/english.go

```diff
@@ -666,6 +667,7 @@ func englishTranslationSet() TranslationSet {
 		LcMergeIntoCurrentBranch:            `merge into currently checked out branch`,
 		ConfirmQuit:                         `Are you sure you want to quit?`,
 		SwitchRepo:                          `switch to a recent repo`,
+		AllBranchesLogGraph:                 `show all branch logs`,
```

Comment:
By convention if we have a string start with a lowecase letter we prefix the key with `Lc` to disambiguate from a similar string with an up-cased first letter

Reply:
updated it https://github.com/jesseduffield/lazygit/pull/1104/commits/8b9f2527947d39a6c52f779a6b3b98e6971cdb02